data/human/human missions.txt

```diff
@@ -5547,3 +5547,221 @@ mission "Quicksilver Mixup 1"
+						not "Blind Man from Martini: label daughter"
+				
+				`	"What happened to your daughter?"`
+					goto illness
+					to display
+						not "Blind Man from Martini: label illness"
+						has "Blind Man from Martini: label daughter"
+
+				`	(Stay silent.)`
+					goto waiting
+					to display
+						not "Blind Man from Martini: label waiting"
+
+			label blind
+			action
+				set "Blind Man from Martini: label blind"
+				"Blind Man from Martini: taxi questions count" ++
+			`	"It was the fourth of July, 3014. I was in my office, on one of the high rises on the outskirts of the capital, with my office window pointed right at the city center." He turns to look out the window, as if he knew New Sydney's skyline was out there where he couldn't see. In the small lull, he takes an audible breath.`
+			`	"It was a beautiful day that day. I... I remember that Pollux was especially bright and colorful that day, and the songbirds that inhabit the city were alive with sound. They were unusually so. It was very vivid, very strange. Maybe I should have listened to the birds...`
+			`	"But when the bomb dropped, I was looking right at it, from my office window. All I could remember next was that I was in a lot of pain, that I couldn't see, but my secretary - thank goodness for her - found me afterwards, during the evacuation, covered in glass from the shockwave. I survived mostly because I was sitting at my terminal and that protected me from the bulk of the shards.`
+				goto taxi
+
+			label blind-1
+			action
+				set "Blind Man from Martini: label blind-1"
+				"Blind Man from Martini: taxi questions count" ++
+			`	"Hmm... well, it was really only one of many things that hit my life then, not least of which was my newfound blindness, and the loss of my company and my job." He gives a heavy sigh, and he shifts in his seat as if something heavy was placed on his shoulders. "I was able to keep things intact for a while, but no, it was too much. I took my life for granted then, but anything can be taken away from you if you get too used to it."`
+				goto taxi
+			
+			label atomic-bombings
+			action
+				set "Blind Man from Martini: label atomic-bombings"
+				"Blind Man from Martini: taxi questions count" ++
+			`	"How do I feel about the bombings? Well, they changed everything. In some ways I'm very grateful that it showed - no, it exposed to me, that people, they- there's people that live this way. With no sight. Who need other people to help them just... live. And that there are plenty of people who don't have people to help them live. I was lucky, I could get treatment for my blindness. But others, no...`
+			`	"And for the bombings themselves, they were certainly a malicious act. The Syndicate needs to take full accountability for it, not only for the people who were killed or affected in some form or another. I'm too cynical to say that they will, with all the power they still have in Parliament."`
```

Comment:
Does checkmate actually expose the Syndicate's involvement?

Reply:
No, but I can use branch to change the dialogue.